### PR TITLE
Add hubble diagrams and data dicts

### PR DESCRIPTION
**Pull request by chowbao**
* Add Hubble data model diagram
* Add Hubble data dictionaries

### CODE REVIEW

File: network/hubble/data-catalog/data-model-diagram.mdx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Data Model Diagram
+sidebar_position: 0
+---
+
+The following diagram shows the relationships between the history and state tables within Hubble.
+
+![Hubble Data Model Diagram](/img/hubble/data_model_diagram.png)
+
+For more detailed information see the [Full Hubble Data Model Diagram](https://dbdiagram.io/d/Hubble-Public-Data-Model-66671e056bc9d447b1501801) or the [Compressed Hubble Data Model Diagram](https://dbdiagram.io/d/Hubble-Public-Data-Model-Compressed-666730996bc9d447b151a181).
```
Comment (sydneynotthecity) [inline network/hubble/data-catalog/data-model-diagram.mdx:10]: Just confirming, these are read only links, correct?

**Review by sydneynotthecity — COMMENTED** [review]
In general, all of the data catalog tables are cut off when viewing the data in the browser. I wonder if we should try and collaps the right hand nav bar on these pages so it's easier to render the full table? Or do you think we should consolidate the information displayed in the table? Most data catalogs can get by with `column_name`, `data_type` and `description`.

File: network/hubble/data-catalog/data-model-diagram.mdx (hunk 1 above)
Comment (chowbao) [reply network/hubble/data-catalog/data-model-diagram.mdx:10]: Yup

File: network/hubble/data-catalog/data-dictionary/enriched-history-operations.mdx
Comment (sydneynotthecity) [inline network/hubble/data-catalog/data-dictionary/enriched-history-operations.mdx:1]: ❓ do you think it's better to repeat the elements documented elements in the here in the `enriched_history_operations` table, or just tell the user it's a combination of ledgers, transactions and operations and show the query that builds the table?

This table in particular is a bit overwhelming to navigate.

**Review by sydneynotthecity — APPROVED** [review]
Definitely good enough for a first pass. There are some improvements that I think we can iterate on:

1. Cleaning up the compressed data model diagram. We need to find a better way to display the entity relationships without the diagram being so cluttered.
2. Pare down the columns in the data dictionaries (or get the tables to render without having to scroll)
3. Adding an overview page that describes table types, and lists all tables + links to each respective dictionary

None of this blocks the merge, I think we should get these out

File: network/hubble/data-catalog/data-dictionary/enriched-history-operations.mdx
Comment (chowbao) [reply network/hubble/data-catalog/data-dictionary/enriched-history-operations.mdx:1]: Yeah I'm not sure 🤔 

From a personal "I don't wanna" perspective, no I don't want to do it cause it is basically copy and pasting something they can click a link to get to.

But sadly for a better user experience I think we should copy and paste

Comment (chowbao) [reply network/hubble/data-catalog/data-dictionary/enriched-history-operations.mdx:1]: I'll make JIRA tickets for this and the above comments

**Pull request merged**